web/src/scenes/actions/common/Form/FieldInput/index.tsx

```diff
@@ -7,26 +7,45 @@ interface FieldInputProps extends InputHTMLAttributes<HTMLInputElement> {
   register: UseFormRegisterReturn;
   errors?: FieldError;
   isDirty?: boolean;
+  value?: string;
+  maxChar?: number; // New prop for maximum characters
 }
 
 export const FieldInput = memo(function FieldInput(props: FieldInputProps) {
-  const { className, register, errors, isDirty, ...otherProps } = props;
+  const {
+    className,
+    register,
+    errors,
+    value = "",
```

Comment:
@andy-t-wang If I'm not mistaken you can get `control` from the `useForm` and pass it to the component. And then use `useWatch` inside the FieldInput component.

Reply:
passing in watch from the call now